README.md

```diff
@@ -110,5 +113,5 @@ Use Javascript to find the element with the `[data-id='3']` attribute, and *remo
 
 **YOUR NOTES**
 ```
-
+Did not have enough time 😕
 ```
```

Comment:
Don't worry! Try again at your earliest convenience and submit. Just let us know when you submit it please, so it's on our radar to give feedback 🙂